src/map/utils/charutils.cpp

```diff
@@ -766,6 +759,37 @@ namespace charutils
         luautils::OnGameIn(PChar, zoning == 1);
     }
 
+    void LoadSpells(CCharEntity* PChar)
+    {
+        // disable all spells
+        size_t length = sizeof(PChar->m_SpellList.data());
+        memset(PChar->m_SpellList.data(), 0, length);
```

Comment:
Actually I would prefer to keep this just so `LoadSpells` can be called any time and the spells are loaded on a "clean slate"